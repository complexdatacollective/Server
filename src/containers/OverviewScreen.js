import React from 'react';

const OverviewScreen = () => (
  <div className="screen">
    <div className="screen__heading">
      <h1 className="screen__heading-title">Network Canvas</h1>
      <h2 className="screen__heading-subtitle">Overview</h2>
    </div>
    <div className="screen__main">
      Various stats and info go here.
    </div>
  </div>
);

export default OverviewScreen;
