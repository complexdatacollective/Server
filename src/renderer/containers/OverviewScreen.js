import React from 'react';
import { OverviewDashboard } from '.';
import { Header } from '../components';

const OverviewScreen = () => (
  <div className="screen">
    <Header />
    <main className="screen__main">
      <OverviewDashboard />
    </main>
  </div>
);

export default OverviewScreen;
