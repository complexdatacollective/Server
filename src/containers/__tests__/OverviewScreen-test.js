/* eslint-env jest */

import React from 'react';
import { shallow } from 'enzyme';
import OverviewScreen from '../OverviewScreen';

describe('<OverviewScreen />', () => {
  it('should render', () => {
    const subject = shallow((
      <OverviewScreen />
    ));

    expect(subject).toMatchSnapshot();
  });
});
