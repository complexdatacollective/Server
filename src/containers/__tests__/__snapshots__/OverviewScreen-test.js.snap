exports[`<OverviewScreen /> should render 1`] = `
ShallowWrapper {
  "complexSelector": ComplexSelector {
    "buildPredicate": [Function],
    "childrenOfNode": [Function],
    "findWhereUnwrapped": [Function],
  },
  "length": 1,
  "node": <div
    className="screen">
    <div
        className="screen__heading">
        <h1
            className="screen__heading-title">
            Network Canvas
        </h1>
        <h2
            className="screen__heading-subtitle">
            Overview
        </h2>
    </div>
    <div
        className="screen__main">
        Various stats and info go here.
        <IPCComponent />
    </div>
</div>,
  "nodes": Array [
    <div
      className="screen">
      <div
            className="screen__heading">
            <h1
                  className="screen__heading-title">
                  Network Canvas
            </h1>
            <h2
                  className="screen__heading-subtitle">
                  Overview
            </h2>
      </div>
      <div
            className="screen__main">
            Various stats and info go here.
            <IPCComponent />
      </div>
</div>,
  ],
  "options": Object {},
  "renderer": ReactShallowRenderer {
    "_instance": ShallowComponentWrapper {
      "_calledComponentWillUnmount": false,
      "_compositeType": 2,
      "_context": Object {},
      "_currentElement": <OverviewScreen />,
      "_debugID": 1,
      "_hostContainerInfo": null,
      "_hostParent": null,
      "_instance": StatelessComponent {
        "_reactInternalInstance": [Circular],
        "context": Object {},
        "props": Object {},
        "refs": Object {},
        "state": null,
        "updater": Object {
          "enqueueCallback": [Function],
          "enqueueCallbackInternal": [Function],
          "enqueueElementInternal": [Function],
          "enqueueForceUpdate": [Function],
          "enqueueReplaceState": [Function],
          "enqueueSetState": [Function],
          "isMounted": [Function],
          "validateCallback": [Function],
        },
      },
      "_mountOrder": 1,
      "_pendingCallbacks": null,
      "_pendingElement": null,
      "_pendingForceUpdate": false,
      "_pendingReplaceState": false,
      "_pendingStateQueue": null,
      "_renderedComponent": NoopInternalComponent {
        "_currentElement": <div
          className="screen">
          <div
                    className="screen__heading">
                    <h1
                              className="screen__heading-title">
                              Network Canvas
                    </h1>
                    <h2
                              className="screen__heading-subtitle">
                              Overview
                    </h2>
          </div>
          <div
                    className="screen__main">
                    Various stats and info go here.
                    <IPCComponent />
          </div>
</div>,
        "_debugID": 2,
        "_renderedOutput": <div
          className="screen">
          <div
                    className="screen__heading">
                    <h1
                              className="screen__heading-title">
                              Network Canvas
                    </h1>
                    <h2
                              className="screen__heading-subtitle">
                              Overview
                    </h2>
          </div>
          <div
                    className="screen__main">
                    Various stats and info go here.
                    <IPCComponent />
          </div>
</div>,
      },
      "_renderedNodeType": 0,
      "_rootNodeID": 0,
      "_topLevelWrapper": null,
      "_updateBatchNumber": null,
      "_warnedAboutRefsInRender": false,
    },
    "getRenderOutput": [Function],
    "render": [Function],
  },
  "root": [Circular],
  "unrendered": <OverviewScreen />,
}
`;
